x-pack/legacy/plugins/siem/public/components/ml/permissions/ml_capabilities_provider.tsx

```diff
@@ -20,12 +21,11 @@ MlCapabilitiesContext.displayName = 'MlCapabilitiesContext';
 
 export const MlCapabilitiesProvider = React.memo<{ children: JSX.Element }>(({ children }) => {
   const [capabilities, setCapabilities] = useState(emptyMlCapabilities);
-  const config = useContext(KibanaConfigContext);
   const [, dispatchToaster] = useStateToaster();
 
   const fetchFunc = async () => {
     try {
-      const mlCapabilities = await getMlCapabilities({ 'kbn-version': config.kbnVersion });
+      const mlCapabilities = await getMlCapabilities({ 'kbn-version': chrome.getXsrfToken() });
```

Comment:
When determining capabilities, does the value of `kbn-version` truly not matter here? I guess it feels weird to be supplying the return value of `getXsrfToken()` as the kbn version. Would you consider adding a comment if this is working as-intended?

Reply:
The `kbn-version` header definitely matters here as if there is a version mis-match you'll receive a `400: Browser client is out of date, please refresh the page` error in return. 

Searches across the code base for `chrome.getXsrfToken()` show it being used for `'kbn-version` all over, from `ml` to `file_uploader`.

Reply:
I am going to try that 'chrome.getInjected('kbnVersion')' suggested by Spencer

Reply:
Fixed it by using the new platform API